### DISCUSSION TITLE
Black Python Devs Mission

### DISCUSSION THREAD

**Discussion by kjaymiller** [BlackPythonDevs/blackpythondevs#10] [General]
Here are the core values: 

> **NOTE**
> This discussion may be updated to reflect any agreed upon changes based on comments in the discussions.

## Secure the Bag: Increase organizational investments in areas with larger densities of Black Python developers globally

Lack of stable employment often prevents members from contributing to the greater community at their desired leve. It is important that Black Python Developers are first and foremost employed and compensated at a fair rate.

## Extend Their Reach: Expand support from existing Python and Other Technology focused organizations to communities of _Black Python Devs_ members .

We can't be everything to everyone. We also don't need to recreate all of the amazing programs designed for Pythonistas. Our goal is to support getting these existing programs to Black developers and fill in the gaps where they exist.

## Support the Ecosystem: Create opportunities to invest in the local communities that members of _Black Python Devs_ represent.

Communities that are void of opportunities are often abandoned. While we can't move entire companies to these areas we can make sure that the local areas and businesses continue to thrive.

## Lend a Hand: Establish guidance, mentorship and support for _Black Python Devs_ Members

To keep an ecosystem alive you must feed it. We want to ensure that the members range in different technical roles, skills, and skill levels so that we can address the widest variety of needs from the community.

**Comment by VeldaKiara** [comment]
Hi @oleksis and @kjaymiller I just checked out the website. Is it possible to move the about us to the home page, seeing that right now its empty and then we can add it when the need arises?
Also, we really need a logo.

**Reply by kjaymiller** [reply]
I think we could definitely have some talking points on the home page... Perhaps do something along the lines of an initial design with a Hero Card and some talking points that match our mission.

**Reply by VeldaKiara** [reply]
That also works, did you talk to the designer I recommended?

**Reply by VeldaKiara** [reply]
Also, do you have highlights of how the talking points would look like?

**Reply by oleksis** [reply]
Any mocks about the Hero card and the talking points?

**Comment by VeldaKiara** [comment]
I also created a banner that we can use for the coffee and code
![coffee and code](https://github.com/BlackPythonDevs/.maintainers/assets/32552296/ece1659a-c167-4309-b30d-8700d0975d3f)
I created it using Canva so there is no issue of infringing or copy rights

**Reply by oleksis** [reply]
I like the banner 👏🏾

**Comment by oleksis** [comment]
> Also, we really need a logo.
 
[Logo discussion](https://github.com/BlackPythonDevs/.maintainers/discussions/12)

**Reply by kjaymiller** [reply]
I think @VeldaKiara is reaching out to another designer about logo/brand work

**Reply by VeldaKiara** [reply]
Yes, I tweeted to them. If I don't get a response, I will find another person too.
